proving/proving.go

```diff
@@ -34,6 +35,8 @@ func Generate(ctx context.Context, ch shared.Challenge, cfg config.Config, logge
 		provingOpts = append(provingOpts, postrs.WithPowCreator(options.powCreatorId))
 	}
 
+	log.Printf("Generating proof with: %d nonces and %d threads.\n", options.nonces, options.threads)
+	logger.Info("proving: start generating proof")
```

Comment:
Nonces and threads can be added directly into the second log.
```suggestion
	logger.Info("proving: start generating proof", zap.Uint("nonces", options.nonces), zap.Uint("threads", options.threads))
```